install/audit-logs.tf

```diff
@@ -1,3 +1,12 @@
+resource "aws_cloudwatch_log_group" "taskcluster_audit_logs" {
+  name = "/aws/kinesisfirehose/taskcluster-staging-audit-logs"
```

Comment:
Is this name specially-formed to say "send all log entries in streams in this log group to the kinesis firehose  source stream named "taskcluster-staging-audit-logs"?

Reply:
This is the name of the cloudwatch logs for the firehose itself which it logs errors to. It was just the name that amazon picked before when I used their ui for it in production so I cargo-culted it here. I will go add comments.